### ISSUE TITLE
Regression with MLP

### ISSUE THREAD

**Issue by darzok** [fchollet/keras#867]
Hello, 

Is it normal that when i use a MLP to make a regression (with time series like sunspot) they are only 'loss' and 'val_loss' that decrease during the training ( acc and val_acc are always at 1.0000) ?

for information, I put a linear activation on output layer (which have only one neural) with mse and each hiden layer have 'relu' activation and dropout

thank you to anybody can give me a good reason :)

**Comment by fchollet** [comment]
Accuracy is not relevant with regression. Simply don't display it (`show_accuracy=False`).

**Comment by darzok** [comment]
Ok thank you very munch a lot for your quick answer :)
